app/models/grda_warehouse/data_source.rb

```diff
@@ -84,6 +85,10 @@ class GrdaWarehouse::DataSource < GrdaWarehouseBase
     where(authoritative: true)
   end
 
+  scope :scannable, -> do
```

Comment:
Did you intend to move it this to `data_source_extension.rb`?

Reply:
I didn't, but I could never get it to work there.